pkg/plugin/runtime_subprocess.go

```diff
@@ -26,8 +27,19 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"helm.sh/helm/v4/pkg/cli"
+	"helm.sh/helm/v4/pkg/plugin/schema"
 )
 
+// Downloaders represents the plugins capability if it can retrieve
+// charts from
+type SubprocessDownloaders struct {
+	// Protocols are the list of schemes from the charts URL.
+	Protocols []string `json:"protocols"`
+	// Command is the executable path with which the plugin performs
+	// the actual download for the corresponding Protocols
+	Command string `json:"command"`
```

Comment:
thinking we should change references to "downloaders" here, since we're changing the plugin type name.

```suggestion
// SubprocessGetterCommands represents a list of subprocess commands for a getter
// plugin, each of which handles one or more protocols defined by the plugin config
type SubprocessGetterCommands struct {
	// Protocols are the list of schemes handled by this command
	Protocols []string `json:"protocols"`
	// Command is the executable path used to handle the protocols
	// the actual download for the corresponding Protocols
	Command string `json:"command"`
```